dev/InfoBar/InfoBar.xaml

```diff
@@ -0,0 +1,201 @@
+                                    </VisualState.Setters>
+                                </VisualState>
+
+                                <VisualState x:Name="Success">
+                                    <VisualState.Setters>
+                                        <Setter Target="ContentRoot.Background" Value="{ThemeResource InfoBarSuccessSeverityStatusBrush}"/>
+                                        <Setter Target="StandardIcon.Text" Value="{StaticResource InfoBarSuccessIconGlyph}"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
+
+                            <VisualStateGroup x:Name="IconStates">
+                                <VisualState x:Name="StandardIconVisible">
+                                    <VisualState.Setters>
+                                        <Setter Target="StandardIcon.Visibility" Value="Visible"/>
+                                        <Setter Target="UserIconBorder.Visibility" Value="Collapsed"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                                
+                                <VisualState x:Name="UserIconVisible">
+                                    <VisualState.Setters>
+                                        <Setter Target="UserIconBorder.Visibility" Value="Visible"/>
+                                        <Setter Target="StandardIcon.Visibility" Value="Collapsed"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                                
+                                <VisualState x:Name="NoIconVisible"/>
+                            </VisualStateGroup>
+
+                            <VisualStateGroup>
+                                <VisualState x:Name="CloseButtonVisible"/>
+                                <VisualState x:Name="CloseButtonCollapsed">
+                                    <VisualState.Setters>
+                                        <Setter Target="CloseButton.Visibility" Value="Collapsed"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
+
+                            <VisualStateGroup x:Name="InfoBarVisibility">
+                                <VisualState x:Name="InfoBarVisible"/>
+                                <VisualState x:Name="InfoBarCollapsed">
+                                    <VisualState.Setters>
+                                        <Setter Target="ContentRoot.Visibility" Value="Collapsed"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
+                            
+                        </VisualStateManager.VisualStateGroups>
+
+                        <Grid
```

Comment:
The grid is not the root of the template to allow devs to set the background property and respect that right? Could we maybe add a comment to explain why? This might seem strange to people trying to retemplate, having a comment explaining would probably help avoid confusion.

Reply:
Right! I will add a comment to that effect.